models.py

```diff
@@ -11,10 +10,11 @@ class ToDo(Base):
     id: Mapped[int] = mapped_column(Integer, primary_key=True, index=True)
     content: Mapped[str] = mapped_column(String)
     session_key: Mapped[str] = mapped_column(String)
+    notes: Mapped[str | None] = mapped_column(Text, nullable=True)  # New column added
 
 
-def create_todo(db: Session, content: str, session_key: str):
```

Comment:
Consider adding a default value for the `notes` field, such as an empty string, to maintain consistency across records.